src/assets/examples/auto-saving.html

```diff
@@ -1,47 +1,48 @@
-<html>
-  <head>
-    <script src="../js/globals.js"></script>
-    <script src="../js/app.js"></script>
-    <link rel="stylesheet" href="../css/app.css">
-    <title>Examples: Autosaving</title>
-  </head>
-  <body>
-    <div id="wrapper">
-    </div>
-    <script>
-      var options = {
-        app: "example-app",
-        autoSaveInterval: 5,
-        mimeType: "text/plain",
-        appName: "CFM_Demo",
-        appVersion: "0.1",
-        appBuildNum: "1",
-        providers: [
-          "localStorage",
-          {
-            "name": "readOnly",
-            "json": {
-              "first-example": "This is the first readonly example",
-              "second-example": "This is the second readonly example"
-            }
-          },
-          {
-            "name": "googleDrive",
-            "mimeType": "text/plain",
-            "clientId": "1095918012594-svs72eqfalasuc4t1p1ps1m8r9b8psso.apps.googleusercontent.com"
-          },
-          {
-            "name": "documentStore",
-            "patch": true
-          }
-        ],
-        ui: {
-          menuBar: {
-            info: "This version auto saves every 5 seconds"
-          }
-        }
-      };
-      CloudFileManager.createFrame(options, "wrapper");
-    </script>
-  </body>
-</html>
+<html>
+  <head>
+    <script src="../js/globals.js"></script>
+    <script src="../js/app.js"></script>
+    <link rel="stylesheet" href="../css/app.css">
+    <title>Examples: Autosaving</title>
+  </head>
+  <body>
+    <div id="wrapper">
+    </div>
+    <script>
+      var options = {
+        app: "example-app",
+        autoSaveInterval: 5,
+        mimeType: "text/plain",
+        appName: "CFM_Demo",
+        appVersion: "0.1",
+        appBuildNum: "1",
+        providers: [
+          "localStorage",
+          {
+            "name": "readOnly",
+            "json": {
+              "first-example": "This is the first readonly example",
+              "second-example": "This is the second readonly example"
+            }
+          },
+          "localFile",
+          {
+            "name": "googleDrive",
+            "mimeType": "text/plain",
+            "clientId": "1095918012594-svs72eqfalasuc4t1p1ps1m8r9b8psso.apps.googleusercontent.com"
+          },
+          {
+            "name": "documentStore",
+            "patch": true
+          }
+        ],
+        ui: {
+          menuBar: {
+            info: "This version auto saves every 5 seconds"
+          }
+        }
+      };
+      CloudFileManager.createFrame(options, "wrapper");
+    </script>
+  </body>
+</html>

```

Comment:
Why does Github think every line has changed here? Is/was there a line-ending issue?